ocean/nemo/nemolite2d/manual_versions/psykal_acc/Makefile.include

```diff
@@ -0,0 +1 @@
+../manual_acc/Makefile.include
```

Comment:
could we point the actual path to the manual_acc version, rather than having a sym link?